cockroachcloud/network-authorization.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Network Authorization
+summary: Learn about the network authorization features for CockroachCloud CockroachDB clusters.
+toc: true
+---
+
+To prevent denial-of-service and brute force password attacks, CockroachCloud requires you to authorize networks that can access your cluster by [allowlisting the public IP addresses](#ip-allowlisting) for your application. Optionally, you can [set up Virtual Private Cloud (VPC) peering](#vpc-peering) for your cluster for enhanced network security and lower network latency.
+
+## IP allowlisting
+
+Authorize your application server’s network and your local machine’s network by [adding their public IP addresses (in the CIDR format) to the CockroachCloud cluster's allowlist](connect-to-your-cluster.html#step-1-authorize-your-network). If you change your location, you will need to authorize the new location’s network, else the connection from that network will be rejected.
+
+- In a development environment, you need to authorize your application server’s network and your local machine’s network. If you change your location, you need to authorize the new location’s network, or else the connection from that network will be rejected.
+- In a production environment, you need to authorize your application server’s network.
+
+{{site.data.alerts.callout_info}}
+While developing and testing your application, you may add `0.0.0.0/0` to the allowlist, which allows all networks. However, before moving into production, make sure you delete the `0.0.0.0/0` network.
+{{site.data.alerts.end}}
+
+If your application servers’ IP addresses are not static or you want to limit your cluster's exposure to the public network, you can connect to your CockroachCloud clusters using VPC Peering instead.
+
+## VPC peering
+
+{{site.data.alerts.callout_info}}
+Self-service VPC peering is a limited-availability feature for GCP clusters. For AWS clusters, [contact us](https://support.cockroachlabs.com/hc/en-us/requests/new).
+{{site.data.alerts.end}}
+
+If you select GCP as your Cloud provider while [creating your CockroachCloud cluster](create-your-cluster.html), you can use [Google Cloud's VPC Network Peering](https://cloud.google.com/vpc/docs/vpc-peering) feature to connect your GCP application directly to your CockroachCloud cluster using internal IP addresses, thus limiting exposure to the public network and reducing network latency.
+
+Setting up a VPC peering connection between your CockroachCloud cluster and GCP application is a two-part process:
+
+- Configure the IP range and size while creating the CockroachCloud cluster
```

Comment:
I didn't realize these were the names of sections below.  In that case, I think these should be links to those sections' headers.